bpm/bonita-core/bonita-login/bonita-login-api/pom.xml

```diff
@@ -23,5 +23,10 @@
 			<artifactId>bonita-session-api</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.bonitasoft.engine.identity</groupId>
+			<artifactId>bonita-identity-api</artifactId>
```

Comment:
Because LoginService#login now throws a SUserNotFoundException which is part of bonita-identity-api